File: Java/FestivaluriClientServer/Common/src/main/java/Model/Angajat.java
package Model;

import java.io.Serializable;

public class Angajat implements IHasId<Integer>, Serializable {

    private Integer id;
    private String username;
    private String password;


    public Angajat(String username, String password) {
        this.username = username;
        this.password = password;

    }

    @Override
    public Integer getId() {
        return id;
    }

    @Override
    public void setId(Integer id) {
        this.id = id;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }



    @Override
    public String toString() {
        return "Angajat{" +
                "id=" + id +
                ", username='" + username + '\'' +
                ", password='" + password + '\'' +
                '}';
    }
}

File: C#/FestivalApp/Model/Angajat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FestivalApp.Model
{
     public class Angajat : IHasID<int>
    {
        public int Id { get; set; }
        public String Username { get; set; }
        public String Password { get; set; }
        public String Email { get; set; }

        public Angajat(string username, string password, string email)
        {
            Username = username;
            this.Password = password;
            this.Email = email;
        }

        public override string ToString()
        {
            return Username + " " + Email;
        }
    }
}
